webdriver/tests/bidi/network/response_completed/response_completed_tentative.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from .. import assert_response_event, HTTP_STATUS_AND_STATUS_TEXT
+
+RESPONSE_COMPLETED_EVENT = "network.responseCompleted"
+
+
+@pytest.mark.parametrize(
+    "status, status_text",
+    [(status, text) for (status, text) in HTTP_STATUS_AND_STATUS_TEXT if status in [101, 407]],
```

Comment:
Is there a GitHub url that we could add here which explains why this is tentative?

Reply:
The reason why it is tentative is that it is not working on Chromium.
"Not working" is typically not a big deal because we can just disable the test / mark expectations as failing, however this cannot be done in this scenario due to https://github.com/web-platform-tests/wpt/issues/42482.

Should I add a comment in the file to say it is blocked on https://github.com/web-platform-tests/wpt/issues/42482?

There's no problem with the tests themselves - my goal is only to unblock our development / CI by disabling two subtests. This is the only workaround I could find - split them into another file.

Reply:
I think we used "tentative" for things still in flux spec-wise (eg spec PR not merged yet). It's fine to split, but maybe find another name? eg `response_completed_status.py`, still with a comment explaining why we had to split this.

Reply:
@jgraham do you have some feedback in why we do not yet support `label` based on a `subtest` metadata entries? I mean there is an issue now but would be good to know why we didn't consider that yet. Thanks.

Reply:
Done